app/assets/stylesheets/application.scss

```diff
@@ -8,6 +8,9 @@ $govuk-new-link-styles: true;
 $govuk-include-default-font-face: false;
 
 @import "govuk_publishing_components/govuk_frontend_support";
+@import "govuk_publishing_components/components/intervention";
```

Comment:
We can remove the import for the intervention component stylesheet as this will automatically be included when the component is used - https://github.com/alphagov/govuk_publishing_components/blob/main/app/views/govuk_publishing_components/components/_intervention.html.erb#L2